test/runnable/test52.d

```diff
@@ -2,28 +2,26 @@
 
 // https://issues.dlang.org/show_bug.cgi?id=2311
 
-extern(C)
+__gshared uint count;
+
+shared static ~this()
 {
-    void exit(int);
-    int printf(const char*, ...);
+    assert(count == 3);
```

Comment:
Why not print the count here? After all this test is about destructors, so it shouldn't rely on them working to ensure test correctness, but if you print the count here the test suite can ensure that it was properly called.

Reply:
Makes sense, will add a printf.